corehq/apps/app_manager/templates/app_manager/v2/partials/form_workflow.html

```diff
@@ -96,25 +94,25 @@ <h4 class="panel-title panel-title-nolink">
             <input type="hidden" name="datums_json" data-bind="value: formLink.serializedDatums"/>
         </div>
         <div class="row">
-            <div class="col-sm-12" data-bind="visible: !formLink.autoLink()">
+            <div class="col-sm-11" data-bind="visible: !formLink.autoLink()">
                 <p>{% trans "This form requires manual linking." %}</p>
             </div>
-            <div class="col-sm-12" data-bind="visible: formLink.autoLink()">
+            <div class="col-sm-11" data-bind="visible: formLink.autoLink()">
                 <p>{% trans "This form doesn't require manual linking." %}</p>
             </div>
-            <div class="col-sm-12" data-bind="visible: formLink.autoLink() && !formLink.manualDatums()">
+            <div class="col-sm-11" data-bind="visible: formLink.autoLink() && !formLink.manualDatums()">
                 <span data-bind="click: enableManualDatums" class="btn btn-default btn-sm">
                     {% trans "Enable manual linking" %}
                 </span>
             </div>
-            <div class="col-sm-12" data-bind="visible: formLink.manualDatums">
+            <div class="col-sm-11" data-bind="visible: formLink.manualDatums">
                 <span data-bind="click: disableManualDatums" class="btn btn-danger btn-sm">
                     {% trans "Disable manual linking" %}
                 </span>
             </div>
         </div>
         <div class="row">
-            <div class="col-sm-12"  data-bind="visible: formLink.showLinkDatums()">
+            <div class="col-sm-11"  data-bind="visible: formLink.showLinkDatums()">
```

Comment:
sm-12 causes the dropdown to spill out of the container (there's extra left margin on a select inside of a .row inside of a .form-group, didn't dig into why).

The alignment still isn't ideal, but I don't think it's any worse than in v1.

v1
<img width="569" alt="screen shot 2017-05-17 at 2 22 31 pm" src="https://cloud.githubusercontent.com/assets/1486591/26169620/709cc8fe-3b0c-11e7-86b9-7612877a5ce2.png">

v2
<img width="510" alt="screen shot 2017-05-17 at 2 22 39 pm" src="https://cloud.githubusercontent.com/assets/1486591/26169615/6c0ee1a0-3b0c-11e7-9ee9-1c7d0344dfc9.png">